src/device-registry/models/Event.js

```
@@ -2947,6 +2947,289 @@ eventSchema.statics.getAirQualityAverages = async function(siteId, next) {
+        $unwind: {
+          path: "$values",
+          preserveNullAndEmptyArrays: false,
+        },
+      },
+
+      // Data quality filtering
+      {
+        $match: {
+          "values.time": { $gte: twoWeeksAgo, $lte: now },
+          "values.pm2_5.value": { $exists: true, $ne: null },
+          // Basic range check for PM2.5 (0-1000 µg/m³ as reasonable limits)
```

Biome (1.9.4)

[error] This property value named values.pm2_5.value is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named values.pm2_5.value


(lint/suspicious/noDuplicateObjectKeys)